apps/web/Dockerfile

```
@@ -11,28 +11,73 @@ ARG NEXT_PUBLIC_CLERK_SIGN_IN_URL=/sign-in
+# SENTRY_AUTH_TOKEN via secret mount (optional): present in CI → withSentryConfig uploads source maps
+# then deletes them; absent (local build) → upload disabled, build still succeeds.
+RUN --mount=type=secret,id=sentry_auth_token,env=SENTRY_AUTH_TOKEN \
+    bun run build:dist \
  && bun run --filter '@aqsha/web' build
 
 FROM node:24-slim AS runtime
 WORKDIR /app
 ENV NODE_ENV=production PORT=3000 HOSTNAME=0.0.0.0 NEXT_TELEMETRY_DISABLED=1
+# Infisical CLI: the entrypoint `infisical run`s the server so runtime secrets (CLERK_SECRET_KEY,
+# SENTRY_DSN_WEB, …) come from Infisical /app. NEXT_PUBLIC_* stay baked at build. Debian → apt repo.
+RUN apt-get update && apt-get install -y --no-install-recommends curl ca-certificates gnupg \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)